update/windows-update.ps1

```diff
@@ -111,23 +116,25 @@ $updatesToDownload = New-Object -ComObject 'Microsoft.Update.UpdateColl'
 $searchResult = $updateSearcher.Search("IsInstalled=0 and Type='Software' and IsHidden=0")
 if ($searchResult.Updates.Count) {
     for ($i = 0; $i -lt $searchResult.Updates.Count; ++$i) {
-        $update = $searchResult.Updates.Item($i)
+        if($i -lt $UpdateLimit){

```

Comment:
instead of this `if`, please add one after `$updatesToDownload.Add($update) | Out-Null` with:

```powershell
if ($updatesToDownload.Count -ge $UpdateLimit) {
    break
}
```